package.json

```diff
@@ -28,8 +28,17 @@
     "sideEffects": false,
     "exports": {
         ".": {
-            "module": "./tslib.es6.js",
-            "import": "./modules/index.js",
+            "module": {
+              "types": "./tslib.d.ts",
+              "default": "./tslib.es6.js"
+            },
+            "import": {
+              "node": "./modules/index.js",
+              "default": {
```

Comment:
This is relying on https://github.com/microsoft/TypeScript/issues/50762 to resolve types for the `node` `import` case, no?